packages/vector_graphics_codec/lib/vector_graphics_codec.dart

```diff
@@ -530,15 +539,80 @@ class VectorGraphicsCodec {
     buffer._currentPathId = -1;
   }
 
+  /// Saves a copy of the current transform and clip on the save stack, and then
+  /// creates a new group which subsequent calls will become a part of. When the
+  /// save stack is later popped, the group will be flattened into a layer and
+  /// have the given `paint`'s [Paint.blendMode] applied.
+  ///
+  /// See also:
+  ///   * [Canvas.saveLayer]
   void writeSaveLayer(VectorGraphicsBuffer buffer, int paint) {
     buffer._putUint8(_saveLayerTag);
     buffer._putUint16(paint);
   }
 
+  /// Pops the current save stack, if there is anything to pop.
+  /// Otherwise, does nothing.
+  ///
+  /// See also:
+  ///   * [Canvas.restore]
   void writeRestoreLayer(VectorGraphicsBuffer buffer) {
     buffer._putUint8(_restoreTag);
   }
 
+  /// Write the [text] contents given starting at [x], [y].
+  ///
+  /// [x], [y] are the coordinates of the starting point of the text baseline.
```

Comment:
`dx` `dy` right?

These shouldn't be `[` because they're not properties.

Reply:
dx/dy are for positioning relative to previous text, x/y are for positioning.

Reply:
oops, I named them wrong below

Reply:
They should be backtick escaped too since they're argument names and not class members

Reply:
no that is incorrect. See all the examples in https://dart.dev/guides/language/effective-dart/documentation#do-start-doc-comments-with-a-single-sentence-summary

Reply:
Haha isn't that doc wrong?  If we generate dartdocs with this the links won't work.

Reply:
it also works in vscode...

Reply:
<img width="1440" alt="Screen Shot 2022-03-31 at 6 51 26 PM" src="https://user-images.githubusercontent.com/8975114/161179194-d695d09b-9df8-4002-8e79-fd2e8f3ee78c.png">

Reply:
Hm. I'm pretty sure I've seen this not work but ok.